community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf

```diff
@@ -212,6 +212,10 @@ variable "nodeset" {
       count = number
       type  = string
     }))
+    dws_flex = optional(object({
```

Comment:
Let's not make it `optional` here. Controller should expected fully populated `nodeset`-entity. It's nodeset-module job to populate while providing user conveniences.